test/built-ins/Temporal/PlainDateTime/prototype/since/no-unnecessary-units.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
This test seems to cover the same things as `returns-days.js` and `units-changed.js`?

Reply:
Right, this seems to be effectively the same thing. I'll kill `units-changed.js`.

Reply:
In that case, we're missing `until/returns-days.js`...

Reply:
It looks like `until/returns-days` got added already in a separate PR: https://github.com/tc39/test262/blob/main/test/built-ins/Temporal/PlainDateTime/prototype/until/returns-days.js . That's what creates the appearance of a mismatch (it exists for `until` but not for `since`).